app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -26,27 +26,160 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-/***** Tooltips *****
+//***** Tooltips *****
 
-.tooltip
+// tooltips used for arbitrary elements
+
+$tooltipBackground: #e3f5ff
```

Comment:
@floriank we're using dashes as separators elsewhere. I'd also suggest using a double dashed `tooltip--` prefix, as we do for selectors.